dev-util/envman/envman-2.1.1.ebuild

```diff
@@ -0,0 +1 @@
+envman-2.1.0.ebuild
```

Comment:
Don't do symlinks. What happens when the previous ebuild gets cleaned from the tree to prefer this one? What if only one needs updating?